lib/einhorn/safe_yaml.rb

```diff
@@ -0,0 +1,17 @@
+require "yaml"
+
+module Einhorn
+  module SafeYAML
+    begin
```

Comment:
could we also check `YAML::VERSION` or YAML.method(:safe_load).parameters` directly instead?

Reply:
Yes, that's another possibility, but I'm too lazy to dig into YAML history to check which version introduced the keyword argument :P 

As for `parameters`, I wouldn't recommend too much, because it can easily be broken if the method is changed to take splat args e.g.`**kargs`. Which often happen for internal refactorings.

Reply:
https://github.com/ruby/psych/commit/0767227051dbddf1f949eef512c174deabf22891

Reply:
No, that's the commit that removed the deprecated positonal arguments, not the one that introduced the new keyword ones.

Reply:
err woops yeah -- and as it turns out it's a bit harrier. Apparently they used to be called `whitelist_classes` at some point (initially):

https://github.com/ruby/psych/commit/4d4439d6d0adfcbd211ea295779315f1baa7dadd#diff-659eac8589abc82c9a0ab3699e4e4be4774d9c09c6c9934af5d9dae0d264439cR322, changed over to `permitted_` here: https://github.com/ruby/psych/commit/682abf20f0ddbc8d83e816022953536dad80c05b